cpp/ql/test/library-tests/syntax-zoo/dataflow-ir-consistency.expected

```diff
@@ -4,7 +4,9 @@ uniqueType
 uniqueNodeLocation
 missingLocation
 uniqueNodeToString
+| cpp11.cpp:50:15:50:16 | (no string representation) | Node should have one toString but has 0. |
```

Comment:
This seems to point to a bug in the dataflow library where we're missing a `tosStringImpl`, which we should probably fix.

Reply:
I have a fix for this locally now, but it pollutes the diff somewhat. This isn't caused by this PR, but instead exposed by this PR. The fix involves a few `+/- 1`s in a few places where global variable modelling meets the funny semantics of array types. I'd like to push the fix to a new PR once this has been merged if that's okay.

Reply:
Fine with me.